### DISCUSSION TITLE
Problem with Confirm Dialog message using tailwind

### DISCUSSION THREAD

**Discussion by F-Andre** [primefaces/community#2057] [PrimeVue]
Hi,
First thanks for your great work.

I've got a problem with Confirm Dialog component to display message using tailwind: style is not applied.

I noticed a problem in the "message" html element

![Capture d’écran du 2024-05-31 08-20-25](https://github.com/primefaces/community/assets/28351867/3890e5ad-36d4-4e9a-ba46-986c884c3888)

I'm using Primevue 3.52, npm 9.8.0, vite 5.2.12.

Is there a way to solve this problem myself ?

Best regards

**Comment by tugcekucukoglu** [comment]
Hi,

Could you share a reproducer link? Did you have any chance to investigate this: https://tailwind.primevue.org/confirmdialog/#presets.lara
